guide/03-the-gis/working-with-different-authentication-schemes.ipynb

```diff
@@ -401,7 +401,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
```

Comment:
@jyaistMap I think we should entirely remove this non-interactive login section as it leads to misplaced expectations.

We can just state 'The recommended suggestion for non-interactive login scripts is to use the built-in identity provider instead of SAML.'

Reply:
@rohitgeo - I made the change you suggested to remove the non-interactive code and section and replaced it with the sentence above.

![image](https://user-images.githubusercontent.com/1399179/80152248-30378e00-8570-11ea-8544-531df0b1efa3.png)